### ISSUE TITLE
fingerprint login support

### ISSUE THREAD

**Issue by timkgh** [pop-os/beta#299]
**How did you upgrade to 21.10? (Fresh install / Upgrade)**
Live USB boot.
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**
Login screen (gdm?)

**Issue/Bug Description:**

Fingerprint login does not seem to be supported.

**Steps to reproduce (if you know):**

**Expected behavior:**

Fingerprint login should work.

**Other Notes:**
Ubuntu 21.10 supports it in its default Gnome desktop, so I know my hardware is supported by Linux.

**Comment by timkgh** [comment]
Yes, it does work with the Ubuntu 21.10 Live USB. I never tried Pop 21.04

This is the fingerprint hardware, on a Lenovo P1 Gen 4.
`Bus 003 Device 003: ID 06cb:00f9 Synaptics, Inc.`

**Comment by kylebakerio** [comment]
I still haven't seen this work in 22.04. I don't even see my fingerprint reader when doing a hardware probe, or in lsusb, weirdly.

I do see a couple reports online from users that supposedly got it working, though, e.g. here: https://www.reddit.com/r/LinuxOnThinkpad/comments/qhoj7v/thinkpad_x1_extreme_gen_4_and_p1_gen_4_how_is/
